config/checkstyle_resources_suppressions.xml

```diff
@@ -74,4 +74,9 @@
   <suppress checks="RegexpSingleline"
     files="[\\/]InputJavadocTagContinuationIndentationGuavaFalsePositive\.java"/>
 
+  <suppress checks="RegexpOnFilename"
+    files="[\\/]SuppressionXpathRegressionPackageAnnotationOne\.java"/>
+  <suppress checks="RegexpOnFilename"
+    files="[\\/]SuppressionXpathRegressionPackageAnnotationTwo\.java"/>
```

Comment:
I don't think this is the best way to handle this. Please provide feedback.

Reply:
in first commit of this PR, please put a `id` property to RegexpOnFilename and name it to explain a reason.
we need to suppress by `id`, not all instances of RegexpOnFilename if few appear later on.

Reply:
Done